apps/transport/lib/jobs/backfill/backfill_resource_history_filesize.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Transport.Jobs.Backfill.ResourceHistoryFileSize do
+  @moduledoc """
+  Backfill of ResourceHistory payload to fill filesize information for all rows.
+  """
+  use Oban.Worker
+  import Ecto.Query
+
+  @backfill_delay 1
+
+  @impl true
+  def perform(%{args: %{"resource_history_id" => resource_history_id, "backfill" => true}}) do
+    with :ok <- perform(%{args: %{"resource_history_id" => resource_history_id}}) do
+      case fetch_next(resource_history_id) do
+        next_id when is_integer(next_id) ->
+          %{resource_history_id: next_id, backfill: true} |> new(schedule_in: @backfill_delay) |> Oban.insert()
+
+        nil ->
+          :ok
+      end
+    end
+  end
+
+  def perform(%{args: %{"resource_history_id" => resource_history_id}}) do
+    update_resource_filesize(resource_history_id)
+    :ok
+  end
+
+  def fetch_next(resource_history_id) do
+    DB.ResourceHistory
+    |> where([rh], rh.id > ^resource_history_id)
```

Comment:
Je pense que tu pourrais faire en sorte d'ignorer ceux qui ont déjà la clé `filesize` de renseignée avec un `not payload ? 'filesize'`. Si jamais on doit le lancer plusieurs fois.

https://stackoverflow.com/a/33631247